### PR TITLE
Managing-volumes: Steps for Replacing brick in gluster volume

### PR DESCRIPTION
**Pull request by pranithk**
Signed-off-by: Pranith Kumar K <pkarampu@redhat.com>

### CODE REVIEW

File: Administrator Guide/Managing Volumes.md [hunk 1]
```diff
@@ -260,38 +261,235 @@ set).
...
-    and now brick 3 is replaced by brick 5.
+**Replacing a brick in *pure* distribute volume**
+
+To replace a brick on a distribute only volume, user needs to remove the brick using remove-brick. Then add a new-brick and rebalance the cluster.
+
+NOTE: Replacing a brick using 'replace-brick' command in gluster is supported only for distributed-replicate or *pure* replicate volumes.
+
+Steps to remove brick Server1:/home/gfs/r2_1 and add Server1:/home/gfs/r2_2:
+
+1. Here is the initial volume configuration:
```
Comment (prashanthpai) [inline Administrator Guide/Managing Volumes.md]: These numbered lists are not rendered properly for some reason.

File: Administrator Guide/Managing Volumes.md [hunk 2]
```diff
@@ -260,38 +261,235 @@ set).
...
+    Bricks:
+    Brick1: Server1:/home/gfs/r2_0
+    Brick2: Server2:/home/gfs/r2_1
+    Brick3: Server1:/home/gfs/r2_2
+    Brick4: Server2:/home/gfs/r2_3
+```
+
+
+Steps:
+0. Make sure there is no data in the new brick Server1:/home/gfs/r2_5
```
Comment (prashanthpai) [inline Administrator Guide/Managing Volumes.md]: Same comment here. These numbered lists are not rendered properly for some reason.

File: Administrator Guide/Managing Volumes.md (hunk 1 above)
Comment (pranithk) [inline Administrator Guide/Managing Volumes.md]: Do you know of any other listing that would render properly?

File: Administrator Guide/Managing Volumes.md (hunk 2 above)
Comment (pranithk) [inline Administrator Guide/Managing Volumes.md]: Do you know of any other listing that would render properly?

**Pull request merged**